ui/src/AccountDropdown/EditContributorsForm.scss

```diff
@@ -65,5 +63,124 @@
       }
     }
   }
+  .inviteViewersLabel{
```

Comment:
I'd leave lots of comments about formatting and using elements vs classes in this file, but Elise and I are on a ticket that will rework a lot of this as soon as it gets merged in, so we can clean it up.